### PR TITLE
Add roadmap for Kraken

### LINKED ISSUE #43: Document the roadmap
We do have issues filed around the features and enhancements which are open/being worked on but it might be useful to have a roadmap in readme with pointers to the major ones like Litmus integration ( https://github.com/openshift-scale/kraken/issues/31 ) for example to help users understand where we need help as well as where the project is going.

### PR DESCRIPTION
**Pull request by chaitanyaenr**
### Description
This commit adds a roadmap which walks through the features and enhancements that
are going to be added to Kraken in the immediate future in order to help users
understand where we need help as well as where the project is going.

### Fixes
#43

### CODE REVIEW

**Comment by yogananth-subramanian** [comment]
Looks good, after we iron out the details network partition scenario outage, proposed by  Alejandro, let's add it to this list.

File: README.md [hunk 1]
```diff
@@ -108,6 +108,15 @@ In addition to checking the recovery and health of the cluster and components un
 - Blog post emphasizing the importance of making Chaos part of Performance and Scale runs to mimic the production environments: https://www.openshift.com/blog/making-chaos-part-of-kubernetes/openshift-performance-and-scalability-tests
 
 
+### Roadmap
+Following is a list of enhancements that we are planning to work on adding support in Kraken. Of course any help/contributions are greatly appreciated.
+- [Ability to visualize the metrics that are being captured by Kraken and stored in Elasticsearch](https://github.com/cloud-bulldozer/kraken/issues/124)
+- Ability to shape the ingress network similar to how Kraken supports [egress traffic shaping](https://github.com/cloud-bulldozer/kraken/blob/master/docs/network_chaos.md) today
+- Continue to improve [Chaos Testing Guide](https://cloud-bulldozer.github.io/kraken/) in terms of adding more best practices, test environment recommendations and scenarios to make sure OpenShift platform as well the applications running on top it are resilient and performant under chaotic conditions
```
Comment (paigerube14) [inline README.md:115]: Think there should be a comma some where in there. 

"...OpenShift platform, as well the applications running on top it, are resilient and performant under chaotic conditions"

Comment (chaitanyaenr) [reply README.md]: Fixed it, thanks.

**Review by paigerube14 — APPROVED** [review]
LGTM

**Pull request merged**